guides/common/modules/proc_adding-custom-deb-repositories-by-using-web-ui.adoc

```diff
@@ -1,10 +1,14 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Adding_Custom_Deb_Repositories_{context}"]
-= Adding Deb repositories
+[id="adding-custom-deb-repositories-by-using-web-ui"]
+= Adding Deb repositories by using {ProjectWebUI}
 
-Use this procedure to add Deb repositories in {Project}.
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-deb-repositories[].
+[role="_abstract"]
+You can add Deb repositories to {Project} by using the {ProjectWebUI}.
+
+.Prerequisites
+* You have extracted and imported the GPG key that you want to use to verify signatures of the Deb packages.
```

Comment:
```suggestion
* You have extracted and imported the GPG key that you want to use to verify signatures of the `Release` files.
```

tech issue: In the Debian/Ubuntu world, packages are generally not signed, just the metadata of repositories, aka. `Release` files.